core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -53,6 +53,18 @@ angular.module('oppia').controller('RteHelperModalController', [
       $scope.modalIsLoading = false;
     });
 
+    // Any form control which is invalid has class name 'ng-invalid'.Using this
+    // property, we can make the form invalid and thus
+    // the Save button can be disabled
+    $scope.disableSave = function() {
+      let disable = false;
```

Comment:
formIsSaveable (and adjust logic accordingly) -- variables should not start with verbs.

Reply:
Done